### PR TITLE
cargo-bench: add page

### PR DESCRIPTION
**Pull request by Cioraz**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/common/cargo-bench.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench --package`
+
+- Benchmark the specified benchmark:
+
+`cargo bench --bench`
+
+- Benchmark with the given profile:
+
+`cargo bench --profile`
```
Comment (acuteenvy) [inline pages/common/cargo-bench.md:24]:
```suggestion
- Benchmark with the given profile (default: `bench`):

`cargo bench --profile {{profile}}`
```

File: pages/common/cargo-bench.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench --no-run`
+
+- Benchmark only the specified packages:
+
+`cargo bench --package`
+
+- Benchmark the specified benchmark:
+
+`cargo bench --bench`
```
Comment (acuteenvy) [inline pages/common/cargo-bench.md:20]:
```suggestion
`cargo bench --bench {{benchmark}}`
```

File: pages/common/cargo-bench.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+`cargo bench`
+
+- Compile, but don’t run benchmarks:
+
+`cargo bench --no-run`
+
+- Benchmark only the specified packages:
+
+`cargo bench --package`
```
Comment (acuteenvy) [inline pages/common/cargo-bench.md:16]:
```suggestion
`cargo bench --package {{package}}`
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**